### ISSUE TITLE
Timezone conversion 

### ISSUE THREAD

**Issue by iogbole** [Appdynamics/apigee-monitoring-extension#13]
Add a timezone conversion feature. 

Apigee SaaS uses UTC tz by default. This results in an error if the extension's host server is using a different timezone. 

**Desired outcome** 

Provide a timezone configuration option in the `config.json`  that accepts either `local` or `utc` as values. 

Where: 

-  `local` indicates that Apigee's timezone on the Apigee extension's host machine is the same as the Apigee server. This caters for on-premise deployment for Apigee. 

-  `utc` instructs the Apigee extension to use `utc` timezone. This caters mostly for Apigee SaaS customers because the SaaS version uses UTC by default. .  When `utc` is set, add  `-u` to all the date functions here - https://github.com/Appdynamics/apigee-monitoring-extension/blob/master/apigee_monitor.sh#L191-#L207 
 
In summary: 

`local` == onpremise Apigee 

```
echo $(date +%m/%d/%Y+%H:%M:%S --date="+1 sec")  > "${timer_file}" 
```

And `utc` == SaaS Apigee 

```
echo $(date -u  +%m/%d/%Y+%H:%M:%S --date="+1 sec")  > "${timer_file}"
```

**Comment by stuartgreenshields** [comment]
I have done the initial first phase of this and it is being put through tests, before being released.

**Comment by stuartgreenshields** [comment]
Completed so in config.json you can select 

1) **onprem** 
This set the APIGEE interval to collect to localtime to where the server runs the script, but this server needs to be in the same local time timezone as the onprem apigee instance you are collecting against.
2) **saas** 
This sets the APIGEE interval to collect to UTC time, wherever the script is run from. This is because the SaaS APIGEE instance time is set to UTC.

**Comment by stuartgreenshields** [comment]
CLosing
